_posts/2023-03-xx-New-release.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: ESMValTool v2.8.0
+date: 2023-03-xx
```

Comment:
"xx" needs to be replace before merging. This also applies to the the filename of this post.